src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,35 @@
+
+import { useEffect, useState } from "react";
+import { getArticleById } from "./api";
+import { useParams } from "react-router-dom";
+
+
+const SingleArticle = () => {
+    const [article, setArticle] = useState([])
+    const {article_id} = useParams();
+
+    useEffect(() => {
+        getArticleById(article_id).then(({data}) => {
+            setArticle(data.articles)
+        })
+
+    }, [article_id])
+
+    return (
+      <section>
+        <h2>Article</h2>
+        <ul>
+          <li key={article.article_id}>
+            <h3>{article.article_title}</h3>
+            <img id="articleImage" src={article.article_img_url}></img>
+            <p>{article.author}</p>
+            <p>{article.created_at}</p>
```

Comment:
This created_at date isn't very readable, you could re-format it to something a bit more readable or use a library like Day.js to do the hard work for you 🙂